modules/ROOT/pages/mutations/create.adoc

```diff
@@ -111,8 +111,8 @@ Read about xref:mutations/update.adoc#_connectorcreate_relationships[`update`] f
 
 == `connectOrCreate` relationships
 
-If a related node has a `@unique` or `@id` directive defined, `connectOrCreate` can be used in a nested `create` to perform a `MERGE` operation on the related node.
-This should create a new relationship and the related node if it doesn't exist yet.
+If a related node has the `@id` and `@unique` directive defined, `connectOrCreate` can be used in a nested `create` to perform a `MERGE` operation on the related node.
+This will create a new relationship and the related node if it doesn't exist yet.
```

Comment:
I don't believe it is a requirement to have both:

```suggestion
If a related node has the `@unique` directive defined, `connectOrCreate` can be used in a nested `create` to perform a `MERGE` operation on the related node.
This will create a new relationship and the related node if it doesn't exist yet.
```